### PR TITLE
restore tests

### PR DESCRIPTION
**Pull request by msokoloff1**

### CODE REVIEW

File: tests/integration/test_client_errors.py [hunk 1]
```diff
@@ -111,21 +110,24 @@ def test_invalid_attribute_error(client, rand_gen):
...
 
-@pytest.mark.slow
-# TODO improve consistency
-@pytest.mark.skip(reason="Inconsistent test")
-def test_api_limit_error(client, rand_gen):
-    project_id = client.create_project(name=rand_gen(str)).uid
+@pytest.mark.skip("timeouts cause failure before rate limit")
+def test_api_limit_error(client):
+    global limited
+    limited = False
```
Comment (gszpak) [inline tests/integration/test_client_errors.py:116]: What is this flag used for?

File: tests/integration/test_client_errors.py [hunk 2]
```diff
@@ -111,21 +110,24 @@ def test_invalid_attribute_error(client, rand_gen):
...
         try:
-            return client.get_project(project_id)
+            return client.get_user()
         except labelbox.exceptions.ApiLimitError as e:
             return e
 
-    with Pool(300) as pool:
-        results = pool.map(get, list(range(2000)))
+    n = 1600
+    with Pool(30) as pool:
```
Comment (gszpak) [inline tests/integration/test_client_errors.py:125]: Just wondering: where do 1600 / 30 come from? It might be worth to leave a comment

**Review by gszpak — APPROVED** [review]
LGTM! Just some nits

File: tests/integration/test_client_errors.py (hunk 1 above)
Comment (msokoloff1) [reply tests/integration/test_client_errors.py:116]: Oops. I'll remove it

File: tests/integration/test_client_errors.py (hunk 2 above)
Comment (msokoloff1) [reply tests/integration/test_client_errors.py:125]: Ok will do

**Pull request merged**